l5kit/l5kit/rasterization/box_rasterizer.py

```diff
@@ -55,14 +54,14 @@ def draw_boxes(
     else:
         im = np.zeros((raster_size[1], raster_size[0], 3), dtype=np.uint8)
 
-    box_world_coords = np.zeros((len(agents), 4, 2))
-    corners_base_coords = np.asarray([[-1, -1], [-1, 1], [1, 1], [1, -1]])
+    corners_base_coords = (np.asarray([[-1, -1], [-1, 1], [1, 1], [1, -1]]) / 2)[None, :, :]
```

Comment:
any reasons to move the `\2`? I'm fine in having it here, just asking :)

Reply:
I think I was trying to reduce the number of operations in each steps. So `corners_base_coords` was outside the `draw_boxes`. But I found it actually don't contribute too much so I move them back into the function. We can put the `/ 2` anywhere if you think that is more readable.

Reply:
I'd move it back to line 60 to make it clear we're using half of the extent. We can also replace it with `*0.5` (mul is generally faster but it won't probably matter here as we don't have thousands of corners)